content/en/docs/application-store/external-apps/deploy-clickhouse.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Deploy ClickHouse Operator and a ClickHouse Cluster on KubeSphere"
+keywords: 'KubeSphere, Kubernetes, ClickHouse, ClickHouse Operator, ClickHouse Cluster'
+description: 'Learn how to deploy ClickHouse Operator and a ClickHouse Cluster on KubeSphere.'
+linkTitle: "Deploy ClickHouse Operator and a ClickHouse Cluster on KubeSphere"
+weight: 14340
+---
+
+[ClickHouse](https://clickhouse.tech/) is a column-oriented database management system (DBMS) for online analytical processing of queries (OLAP). ClickHouse on QingCloud is a ClickHouse cluster application deeply customized with the following features.
+
+- Automated cluster management. ClickHouse on QingCloud manages cluster status via Paxos protocols. As you run ClickHouse on QingCloud and manage it like a single-node ClickHouse cluster, you will be free from complicated cluster management operations.
+- Data redistribution in clusters. Apart from automated management of the cluster, you can also perform cluster-level data operations, including `SYSTEM RESHARDING`, `SYSTEM RESHARDING DATABASE [database_name]`, and `SYSTEM RESHARDING TABLE [database_name].[table_name]`. Since you can redistribute your cluster data based on you needs, data query performance will be evidently optimized with less scenarios like inappropriate data distributions.
```

Comment:
data query performance will be evidently optimized with less scenarios like inappropriate data distributions.
->
data query performance will be greatly optimized.